blog/how_to_write_unit_testing_for_smart_contracts_with_truffle.md

```diff
@@ -0,0 +1,471 @@
+# How to Write Unit Testing for Smart Contracts with Truffle
+
+# Introduction
+When creating decentralized applications that leverage smart contracts, it is important to ensure that little or no vulnerabilities to prevent an attacker from compromising your application.
```

Comment:
```suggestion
When creating decentralized applications that leverage smart contracts, it is important to ensure that there are little or no vulnerabilities to prevent an attacker from compromising your application.
```